app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -570,21 +574,30 @@ class MainActivity : AppCompatActivity(), ReplayModeFragment.OnSelectModeItemLis
         VisionReplayManager.destroy()
     }
 
-    private fun showReplayModeFragment() {
-        supportFragmentManager
+    private fun showReplayModeFragment(stateLoss: Boolean = false) {
```

Comment:
I guess for bool description you meant `stateLost`.

Reply:
It has the same association with commitWith**StateLoss**(), I'd stay stateLoss